social-network/public/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#000000">
+    <!--
+      manifest.json provides metadata used when your web app is added to the
+      homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
+    -->
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" type="text/css" href="https://cdn-static-1.medium.com/_/fp/css/main-branding-base.X-Nw8EZw3j7tl3MaFDQL1g.css">
```

Comment:
While this takes care of the styling, you pretty much copied Medium's DOM layout and stylesheet. The intent of the assignment is to not copy the code, but to write your own. Try again without doing this.